virtcontainers/cgroups_test.go

```diff
@@ -134,6 +134,7 @@ func TestUpdateCgroups(t *testing.T) {
 
 	// path doesn't exist
 	s.state.CgroupPath = "/abc/123/rgb"
+	s.config = &SandboxConfig{}
```

Comment:
Was this the only of failure for s.config :) ? 

Is it feasible to add a unit test to cover the case we are adding here, podCgroupOnly?